_config.yml

```diff
@@ -298,3 +299,18 @@ defaults:
       type: "pages"
     values:
       layout: docs-v2
+t:
+  blog:
+    gb: "Blog"
```

Comment:
`gb` should be `en` for all occurrences below

Reply:
as of country/language codes I have a table published there https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2

With such a naming convention we should use English flag rather that GB.

in case of Brazil-Portuguese we would go for `br` rather that for `pt`.

Also, the `gb` goes nicely with `famfamfam` flag naming 

Shall I change it anyway?

Reply:
I think we should change this to `en`. See https://en.wikipedia.org/wiki/ISO_639-1. This is a language and not a region (country) after all.